examples/DictionaryExample/Program.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Momento.Sdk.Exceptions;
+using Momento.Sdk.Incubating;
+
+public class Driver
+{
+    private static readonly string AUTH_TOKEN_ENV_VAR = "TEST_AUTH_TOKEN";
+    private static readonly string CACHE_NAME_ENV_VAR = "TEST_CACHE_NAME";
+
+    public async static Task Main()
+    {
+        var authToken = ReadAuthToken();
+        var cacheName = ReadCacheName();
+
+        using var client = Momento.Sdk.Incubating.SimpleCacheClientFactory.CreateClient(authToken, 60);
```

Comment:
I still don't trust this form of `using`.  I feel like I've seen it break several times.  It's possible it's FUD.  Either way it would be good to be consistent between the examples.  Not a blocker.

Also whenever Pete's auth token changes land we will need to update this, and we should probably wait until that lands before sending stuff over to Tom so he doesn't have to update twice.